torchrecipes/audio/source_separation/conf/train_app.yaml

```diff
@@ -0,0 +1,16 @@
+_target_: torchrecipes.audio.source_separation.train_app.SourceSeparationTrainApp
+
+defaults:
+  - _self_
+  # module
+  # - schema/module: conv_tasnet_module_conf
+  # datamodule
+  - schema/datamodule: librimix_datamodule_conf
+  - datamodule: librimix
+  # trainer
+  - schema/trainer: trainer
+  - trainer: gpu
+
+hydra:
+  searchpath:
+    - pkg://torchrecipes.core.conf
```

Comment:
We might not this search path as `gpu.yaml` is included in `torchrecipes/audio/source_separation/conf`.

Reply:
For the unit test, I'm not sure if we need to override the trainer config with some CPU options. If we already have GPUs for CI, then we probably don't have to.

Reply:
Sure. let me change the test config to cpu.